cves/CVE-2013-0863.yml

```diff
@@ -64,7 +64,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: Buffer overflow in the rle_decode functin in libavcode/sanm.c in FFmpg before 1.0.4
```

Comment:
This is probably why the yaml is failing - use the pipe character for multiline strings

Reply:
This is just a copy and paste of the NVD. Please don't plagiarize and fix this immediately.